index.bs

```diff
@@ -2825,8 +2825,8 @@ PDF document represented as a Base64-encoded string.
       }
 
       browsingContext.PrintPageParameters = {
-        ? height: (float .ge 0.0) .default 27.94,
-        ? width: (float .ge 0.0) .default 21.59,
+        ? height: (float .ge (2.54 / 72)) .default 27.94,
+        ? width: (float .ge (2.54 / 72)) .default 21.59,
```

Comment:
Unfortunately this is a repeating decimal, so I opted for the fraction representation instead. We could also round up (but not down), if preferred.